ebms-send-in/src/main/kotlin/no/nav/emottak/ebms/App.kt

```diff
@@ -39,18 +44,27 @@ suspend fun ResourceScope.setupServer() {
 
     val prometheusMeterRegistry = PrometheusMeterRegistry(PrometheusConfig.DEFAULT)
 
+    val kafkaPublisherClient = EventPublisherClient(config().kafka)
+    val eventLoggingService = EventLoggingService(kafkaPublisherClient)
+
+    val eventRegistrationScope = this.coroutineScope(Dispatchers.IO)
```

Comment:
Du kan fjerne `this` siden `coroutineScope()` bare er tilgjengelig i et` ResourceScope`